tests/compile-fail/custom_crate_omitted.rs

```diff
@@ -0,0 +1,9 @@
+mod darling {}
+
+#[derive(darling::FromDeriveInput)]
+// forgot to specify #[darling(crate = darling)]
+pub struct Example {
+    ident: Option<syn::Ident>,
+}
+
+fn main() {}
```

Comment:
```suggestion
use ::darling as not_darling;

mod darling {}

#[derive(not_darling::FromDeriveInput)]
// forgot to specify #[darling(crate = not_darling)]
pub struct Example {
    ident: Option<syn::Ident>,
}

fn main() {}
```

As written, this test isn't showing what happens when the macro is actually invoked and tries to use the default `darling` path only to have that not work.

Reply:
Huh. Okay, in that case let's just delete this test.